crates/bevy_ui/src/focus.rs

```diff
@@ -95,8 +95,7 @@ impl RelativeCursorPosition {
     /// A helper function to check if the mouse is over the node
     pub fn mouse_over(&self) -> bool {
         self.normalized
-            .map(|position| self.normalized_visible_node_rect.contains(position))
-            .unwrap_or(false)
+            .map_or(false, |position| self.normalized_visible_node_rect.contains(position))
```

Comment:
there's some distaste amongst some contributors (me included) with using `map_or` as it's harder to read because it put the "or" case first